Packages/com.llealloo.audiolink/Editor/Scripts/AudioLinkConfiguratorEditor.cs

```diff
@@ -8,7 +8,7 @@
 #else
   using UnityEditor.Experimental.SceneManagement;
 #endif
-#if UDON
+#if UDONSHARP
```

Comment:
I don't remember this define. Can you tell me when it is defined? I only remember UDON, COMPILER_UDONSHARP and UNITY_EDITOR really

Reply:
UDONSHARP is sorta new and it's set by U#, this is probably unnecessary with VPM but it's more correct  than UDON for our usecase because UDON can be true without U# being imported into the project

Reply:
Set during compilation or just set when U# is imported?

Reply:
when U# is imported